pkg/fanal/secret/builtin-rules.go

```diff
@@ -792,4 +793,13 @@ var builtinRules = []Rule{
 		SecretGroupName: "secret",
 		Keywords:        []string{"typeform"},
 	},
+	{
+		ID:              "dockerconfig-secret",
+		Category:        CategoryDocker,
+		Title:           "Dockerconfig secret exposed",
+		Severity:        "HIGH",
+		Regex:           MustCompile(`(?i)(\.(dockerconfigjson|dockercfg):\s*\|*\s*(ey|ew)+)`),
+		SecretGroupName: "secret",
+		Keywords:        []string{"dockerconfig"},
```

Comment:
So as not to miss `dockercfg: ***` secrets.
```suggestion
		Keywords:        []string{"dockerc"},
```